FEATURE_MATRIX.md

```diff
@@ -0,0 +1,43 @@
+
+**SIG Content**
+
+| Subsystem | Detail | Feature    | Code | Release | Link | Doc | 
+|:---------|:-------------|:--------|:-------|:------------|:-----------:|:-------:|
+|   Frameworks    ||||||| 
+|           | AzToolsFramework | 🟢 Stable | 🟢 Stable | 🟢 Stable |||
+|           | Lua Integration  | 🟢 Stable | 🟢 Stable | 🟢 Stable || [Doc](https://o3de.org/docs/user-guide/scripting/lua/) |
+|           | Prefabs          | 🟢 Stable | 🟢 Stable | 🟢 Stable ||| 
+|           | Qt for Python    | 🟢 Stable | 🟢 Stable | 🟢 Stable ||| 
```

Comment:
Should this also include platform support? Some functionality may still not be available on Linux or Mac.

Reply:
I'll review what other sigs do and follow suit